tetrateci/test_1.9.sh

```diff
@@ -1,6 +1,9 @@
 #!env bash
 set -e
 
+# istio 1.9 is not supported for k8s 1.16
```

Comment:
We need to get this logic into create_cluster script itself so that we don't create 1.16 cluster for 1.9 Istio and similarly 1.19 cluster to test 1.7Istio.

Is there a better way to do it otherwise?

Reply:
thought about that but the problem is, creation and deletion is split into multiple scripts would be difficult to maintain later